dbt/include/teradata/macros/adapters.sql

```diff
@@ -178,31 +193,42 @@
       NULL AS table_database,
       ColumnsV.DatabaseName AS table_schema,
       ColumnsV.TableName AS table_name,
+      {%- if table_kind == 'V ' -%}
+        'view' AS table_type,
+      {%- else -%}
       CASE WHEN TablesV.TableKind = 'T' THEN 'table'
         WHEN TablesV.TableKind = 'O' THEN 'table'
         WHEN TablesV.TableKind = 'V' THEN 'view'
```

Comment:
Do we require TablesV.TableKind = 'V' THEN 'view' ?

Reply:
Yes, this is not required.